libia2/include/ia2.h

```diff
@@ -72,8 +72,10 @@ asm(".macro mov_pkru_eax pkey\n"
 
 #ifdef LIBIA2_INSECURE
 #define protect_tls_for_compartment(n)
+#define late_protect_tls_for_compartment(n)
```

Comment:
I think the `late_` name leaves something to be desired--this is called later than `protect_tls_for_compartment(n)` but specifically it's called on new thread creation, not just any time later. `late_` doesn't specify an invariant or explain specifically what is done differently (i.e., the name doesn't convey that the difference is that `late_init_tls_N` is wrapped in call-gates).